scripts/obico.sh

```diff
@@ -0,0 +1,433 @@
+#!/usr/bin/env bash
+
+#=======================================================================#
+# Copyright (C) 2020 - 2022 Dominik Willner <th33xitus@gmail.com>       #
+#                                                                       #
+# This file is part of KIAUH - Klipper Installation And Update Helper   #
+# https://github.com/th33xitus/kiauh                                    #
+#                                                                       #
+# This file may be distributed under the terms of the GNU GPLv3 license #
+#=======================================================================#
+
+set -e
+
+#===================================================#
+#============== INSTALL MOONRAKER-OBICO ============#
+#===================================================#
+
+function moonraker_obico_systemd() {
+  local services
+  services=$(find "${SYSTEMD}" -maxdepth 1 -regextype posix-extended -regex "${SYSTEMD}/moonraker-obico(-[0-9a-zA-Z]+)?.service")
+  echo "${services}"
+}
+
+function moonraker_obico_config() {
+  local moonraker_cfg_dirs=($(get_config_folders))
+  if [[ -n "${moonraker_cfg_dirs}" ]]; then
+    echo "${moonraker_cfg_dirs[${i}]}/moonraker-obico.cfg"
+  else
+    echo ""
+  fi
+}
+
+function moonraker_obico_needs_linking() {
+  moonraker_obico_cfg=${1}
```

Comment:
Can this variable be local? Or is it required to be global? I see it only being used inside the `moonraker_obico_needs_linking` function